student-info-system/professor/tests/tests_views.py

```diff
@@ -16,9 +16,11 @@ def setUpTestData(self):
                                       first_name='p',
                                       last_name='rof',
                                       password='hello')
+        u2 = User.objects.create_user(username='u2', first_name='s', password='hello')
         profile = Profile.objects.create(user=u1, role='P')
+        sprofile = Profile.objects.create(user=u2, role='S')
         prof = Professor.objects.create(profile=profile, major=major)
-
+        stud = Student.objects.create(profile=sprofile, major=major)
```

Comment:
can you use createStudent, etc? that'll isolate the changes

Reply:
done